validator/testdata/feature_tests/amp_sidebar.out

```diff
@@ -0,0 +1,4 @@
+FAIL
```

Comment:
Yes, but not in this change. I've started by just splitting validator.protoascii for now and will get the tests done seperately.